app/footer.tsx

```diff
@@ -1,3 +1,200 @@
+import Image from 'next/image';
+import { CiGlobe } from 'react-icons/ci';
+import { FaFacebookSquare, FaLinkedinIn, FaTwitter } from 'react-icons/fa';
+import { MdMailOutline, MdOutlineCall } from 'react-icons/md';
+import { SlLocationPin } from 'react-icons/sl';
+
+const quickLinks = [
+  {
+    text: 'Telephone Directory',
+    link: '',
+  },
+  {
+    text: 'Books and e-library',
+    link: '',
+  },
+  {
+    text: 'Important Links',
+    link: '',
+  },
+  {
+    text: 'Downloads',
+    link: '',
+  },
+  {
+    text: 'RTI',
+    link: '',
+  },
+  {
+    text: 'NIT KKR IT Infrastructure Usage Policy',
+    link: '',
+  },
+  {
+    text: 'NIT KKR @NDL',
+    link: '',
+  },
+  {
+    text: 'Telephone Directory',
+    link: '',
+  },
+];
+
+const aboutUsItems = [
+  { text: 'Council of NITs', link: '' },
+  { text: 'NAD Digilocker', link: '' },
+  { text: 'NIRF (Data & Certificate)', link: '' },
+  { text: 'NBA Accreditation Status', link: '' },
+  { text: 'ARIIA', link: '' },
+  { text: 'Skill Hub (PMKVY 4.0)', link: '' },
+  { text: 'Jobs @ NIT KKR', link: '' },
+];
+
+const departmentsItems = [
+  { text: 'Vigilance Corner', link: '' },
+  { text: 'PRO', link: '' },
+  { text: 'OBC & PWD Cell', link: '' },
+  { text: 'SC/ST Cell', link: '' },
+  { text: 'Join Alumni Association', link: '' },
+  { text: 'User Login', link: '' },
+  { text: 'Contact Us', link: '' },
+];
+
 export default function Footer() {
-  return <></>;
+  return (
+    <>
+      <div className="flex flex-col justify-center bg-[#000] md:flex-row lg:items-center">
```

Comment:
Will not merge any div tags. Please refer https://developer.mozilla.org/en-US/docs/Web/HTML/Element and update to the most relevant semantic tags.